udata/core/dataset/rdf.py

```diff
@@ -468,40 +469,29 @@ def title_from_rdf(rdf, url):
             return i18n._("Nameless resource")
 
 
-def access_rights_from_rdf(resource: RdfResource):
+def access_rights_from_rdf(resource: RdfResource) -> str | None:
     """
     Extract the access rights from a RdfResource
-    If the access rights is a URIRef, return it as a string.
-    If the access rights is a RdfResource, return the label of the RdfResource if any, or the identifier of the RdfResource.
+    Cardinality is 0..1.
```

Comment:
Yes... but no...
All `otherRestrictions` linked to an `accessConstraints` will end up here, so we unfortunately need 0..n. At least until we move to SEMIC v3.

Reply:
Yep, will change.